### DISCUSSION TITLE
Option to turn down the crowd noise / peep screaming in the Audio Settings.

### DISCUSSION THREAD

**Discussion by Shima33** [OpenRCT2/OpenRCT2#17109] [Feature requests]
Please. I'm 200 years into my save. My ears hurt.

**Comment by ocalhoun6** [comment]
The peep noises are the entire reason my girlfriend refuses to even try playing this game. She hates them so much.

**Comment by Derpercoaster** [comment]
I agree. It gets obnoxious quickly. I often place a merry-go-round in a quiet corner of the park, and turn on the "listen to the ride" thing, and just shove the ride window to the bottom of the screen. No more screaming, just peaceful organ music.

**Comment by karst** [comment]
this should now be possible using asset packs. You can create an asset pack with the same files but lower volume, by for example lowering it on the master in a DAW. I do not remember if there's a json tag to lower the volume yet.

**Comment by ZehMatt** [comment]
I do think that adding a new audio channel for peeps is a reasonable request.

Feature request is now tracked on https://github.com/OpenRCT2/OpenRCT2/issues/19712
